packages/@react-native-windows/tester/src/js/utils/RNTesterList.windows.js

```diff
@@ -26,9 +26,24 @@ const ComponentExamples: Array<RNTesterExample> = [
     module: require('../examples/FlatList/FlatListExample'),
   },
   {
-    key: 'DatePickerExample',
-    category: 'UI',
-    module: require('../examples-win/DatePicker/DatePickerExample'),
```

Comment:
Unintentionally removed?

Reply:
Removed on purpose - this page is not in core, and there's no need to demo a deprecated module - we have the community implementation, showcased in the Gallery app.

Reply:
If we haven't removed the module yet, I'm not sure we should remove the example page and disable the test to visit it. Esp with lean-core in an indeterminate state.

If we did want to remove this though, we should also remove the `DatePickerExample` file this references,

Reply:
Since the example is not in core, but the module still is, I'd say let's align with core - so I'd remove the example file altogether, which I forgot. 
I don't feel too strongly about this, though.

Reply:
Philosophically I think test coverage is good to have for anything we're offering support for. I think those benefits might outweigh loss of alignment.

I refreshed myself a bit on `DatePicker`. I had forgotten that upstream had `DatePickerIOS` and `DatePickerAndroid`, where we expose our own custom `DatePicker` (not exposed by core). We've had it yellow-boxing in at least 0.64, and it doesn't have a pure upstream equivalent, so we could actually probably fully remove it now.

Reply:
Going to restore the example for now. The whole module and its references can be removed later on.